src/styles/blocks/cardMain.scss

```
@@ -0,0 +1,85 @@
+.card {
+  width: 200px;
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  margin: 0 auto;
+  align-items: center;
+  border: 1px solid $color_text_secondary;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  vertical-align: baseline;
+
+   &__img {
+    width: 160px;
+    height: 134px;
+    background-image: url('../images/imac.jpeg');
+    background-size: cover;
+   }
```

```suggestion
   &__img {
     width: 160px;
     height: 134px;
     background-image: url('../images/imac.jpeg');
     background-size: cover;
   }
```
there should be correct indentations according to style rules